src/encoded/commands/deploy.py

```diff
@@ -812,6 +812,7 @@ def main():
         # helps vars for release and building amis
         for helper_var in helper_vars:
             print(helper_var)
+    print('Done')
```

Comment:
Got tired of AWS leaving you hanging?

Reply:
The previous print line would not print...I think it has to do with my terminal config but printing Done seems okay.